setup.cfg

```diff
@@ -96,12 +92,8 @@ tracing =
     opentelemetry-exporter-zipkin
 
 [options.package_data]
-bentoml =
-    py.typed
-    bentoml/*
-bentoml_cli =
-    py.typed
-    bentoml_cli/*
+bentoml = bentoml/*
+bentoml_cli = bentoml_cli/*
```

Comment:
This is to fix with newer version of setuptools that doesn't add package_data automatically.